package.json

```diff
@@ -85,7 +85,7 @@
     "chai-subset": "^1.6.0",
     "cross-env": "^7.0.3",
     "eslint": "7.27.0",
-    "electron": "15.3.0",
+    "electron": "19.0.8",
```

Comment:
Intentional?

Reply:
Yeah, this is the version used for unit tests. If we don't keep this updated we can get dependabot security alerts.

This does mean we only run the unit tests in one Electron version but the e2e tests should keep on top of any breakages between Electron versions.